ofl/yomogi/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1,12 @@
+<p>
+Yomogi is extra thin hand writing font that is easy to read and makes a strong impression. It includes Google Latin Plus, hiragana, katakana, JIS level 1 and 2 kanji glyphs. 
+</p>
+<p>
+The design of the letters combined cuteness and readability, and it is made into a monospaced font that could also be used in the vertical writing style.
+</p>
+<p>
+This font is licensed under the SIL Open Font License, Version 1.1.
```

Comment:
Since almost all GF fonts are under OFL, we can also remove this line.

Reply:
Done.